### PR TITLE
[DEV-3217] financeLog pending Amounts

### PR DESCRIPTION
**Pull request by Yannick1712**
#### Pre-Release

- [ ] Check migrations
  - No database related infos (sqldb-xxx)
  - Impact on GS (new/removed columns)
- [ ] Check for linter errors (in PR)
- [ ] Test basic user operations (on [DFX services](https://dev.services.dfx.swiss))
  - Login/logout
  - Buy/sell payment request
  - KYC page

#### Post-Release

- Test basic user operations
- Monitor application insights log

### CODE REVIEW

File: src/subdomains/supporting/log/log-job.service.ts [hunk 1]
```diff
@@ -87,6 +93,22 @@ export class LogJobService {
...
+      BankTxType.KRAKEN,
+    );
+    const pendingKrakenTx = await this.exchangeTxService
+      .getPendingExchangeTx(
+        ExchangeTxType.WITHDRAWAL,
+        ExchangeName.KRAKEN,
+        'Maerki Baumann & Co. AG',
+        'Bank Frick (SEPA)',
+      )
+      .then((p) => p.filter((p) => p.currency === 'EUR'));
```
Comment (davidleomay) [inline src/subdomains/supporting/log/log-job.service.ts:111]: Maybe this methods should be renamed to `getRecent...`, they are not pending 😉

File: src/integration/exchange/entities/exchange-tx.entity.ts
Comment (davidleomay) [inline src/integration/exchange/entities/exchange-tx.entity.ts:1]: Why these changes? 🤔

**Review by davidleomay — COMMENTED** [review]
- [x] Get last 7 days of receiver and 14 days of sender transactions (instead of fixed IDs)

File: src/integration/exchange/entities/exchange-tx.entity.ts
Comment (Yannick1712) [reply src/integration/exchange/entities/exchange-tx.entity.ts:1]: It's not working with Entity Methods these way. With entity methods you would have to implement these on every position you used the dto 😅

Comment (davidleomay) [reply src/integration/exchange/entities/exchange-tx.entity.ts:1]: You can just use `export type ExchangeTxDto = Omit<ExchangeTx, keyof IEntity | 'pendingBankAmount'>;`

Comment (Yannick1712) [reply src/integration/exchange/entities/exchange-tx.entity.ts:1]: Yeah sure, but that's a pain with every entity method we will add 🤔 Thought about this solution too but for me this solution isn't so nice, instead of just creating a real dto 🤔 but yeah I we can do this. 
Yeah with the extra dto class we don't need the `?`, because they are defined in the dto itself but clearly if we change that we have to add them in the entity again 👍

**Review by davidleomay — APPROVED** [review]
